docs/cli.md

```diff
@@ -103,5 +103,29 @@ To:
 ERROR [2016-03-09T15:27:09.339Z] (44127 on MacBook-Pro-3.home): Error message.
     Stack of the error
 statusCode: 500
-v: 1
-```
\ No newline at end of file
+```
+
+In order to print all nested properties of `Error` objects, you can use `--errorProps` flag with `*` property.
+
+Please note, that you need to escape `*` character (`'*'` or `\*`), because `*` is a wildcard in Bash.
```

Comment:
I think this can be struck completely. Either that, or rephrase to: "Note: you must quote or escape the `*` (asterisk) to avoid shell expansion."

Reply:
changed to 
> "Note: you must quote or escape the * (asterisk) to avoid shell expansion."